aya_tamimi.py

```diff
@@ -0,0 +1,38 @@
+books = [ {
+    "title": "Python Fundamentals",
+    "author": "Ahmed Zayed",
+    "availability" : True,
+    "tags": ["Programming", "Python", "Basics"]
+},
+{
+    "title": "The Art of Coding",
+    "author": "Mohammed Ali",
+    "availability" : False,
+    "tags": ["Coding", "Art", "Creativity"]
+},
+{
+    "title": "Data Structures in Python",
+    "author": "Yousef Amr",
+    "availability" : True,
+    "tags":["Data", "Python", "Intermediate"]
+}
+]
+
+# Task one Filter Books by Author and availability
+available_books_by_mohammed = []
+for book in books:
+        if book['author'] == "Mohammed" and book['availability'] == True :
```

Comment:
We don't need the book authors to be exactly "Mohammed," but we need all books where the author's name contains "Mohammed."

For example, if author = "Mohammed Ali" and availability = True, we should add the book to available_books_by_mohammed.